spec/models/file_type_filter_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe FileTypeFilter, type: :model do
+  describe 'Validations' do
+    it 'is valid with a label when enabled' do
+      file_type_filter = described_class.new(label: 'Some label', enabled: true)
+      expect(file_type_filter).to be_valid
+    end
+
+    it 'is invalid without a label when enabled' do
```

Comment:
I though the ticket said that when enabled, if the label is not set it should use the default value, so the filter can be enabled without a label, but I'll show up the default value in the frontend.

The issue I see now is not having default values for filters. Label should not be need to be set in order to use the filter AFIK.